preprocessing/data-import/consolidate-csv.js

```diff
@@ -0,0 +1,70 @@
+// converts the output of rdata-to-csv.r
```

Comment:
converts it into what?

Reply:
Comment have been extended in commit 09b8135 to further explain what it converts too.
Resolving conversation.